packages/build-info/src/frameworks/nuxt.ts

```diff
@@ -28,18 +28,16 @@ export class Nuxt extends BaseFramework implements Framework {
     await super.detect()
 
     if (this.detected) {
-      if (this.isV3(this.detected)) {
+      // Use Nuxt 3+ config as default, only apply legacy config for versions < 3
```

Comment:
what I meant by "use nuxt 3+ as default" was that I want this PR to change the config on lines 9 through 19 to the CURRENT NUXT 3+ config. then the dynamic `detect()` stuff should only override legacy nuxt <=2 config as necessary.